doc: tm_strcontains documentation. (#87)
terramate-io/terramate-docs: cli/reference/functions/tm_strcontains.md

---
title: tm_strcontains | Functions | Configuration Language
description: The tm_strcontains function checks whether a substring is within another string.
---

# `tm_strcontains` Function

> `tm_strcontains` function checks whether a substring is within another string.

Reviewer: ```suggestion
The `tm_strcontains` function checks if a given string contains a specific substring.
```

```
tm_strcontains(string, substr)
```

## Examples

```sh
tm_strcontains("hello world", "wor")
true

tm_strcontains("hello world", "wod")
false
```

## Related Functions

> * [`tm_startswith`](./tm_startswith.md) takes two values: a string to check and a prefix string. The function returns true if the string begins with that exact prefix.

Reviewer: ```suggestion
* [`tm_startswith`](./tm_startswith.md) checks if a string starts with a specified prefix. It takes two arguments: the string and the prefix, returning true if the string starts with the exact prefix.
```

> * [`tm_endswith`](./tm_endswith.md) takes two values: a string to check and a suffix string. The function returns true if the first string ends with that exact suffix.

Reviewer: ```suggestion
* [`tm_endswith`](./tm_endswith.md) checks if a string ends with a specified suffix. It takes two arguments: the string and the suffix, returning true if the string ends with the exact suffix.
```
